test/unit/offset.js

```diff
@@ -548,7 +548,7 @@ QUnit.test( "fractions (see #7730 and #7885)", function( assert ) {
 
 	result = div.offset();
 
-	assert.equal( result.top, expected.top, "Check top" );
+	assert.equal( Math.abs( result.top - expected.top ) < 0.25, true, "Check top within 0.25 of expected" );
```

Comment:
Since we only check within a range for top, not left, it'd be good to have a support comment, like:

    // Support: Chrome 45-46+
    // In recent Chrome those values differ a little.